controllers/user.controller.js

```diff
@@ -0,0 +1,40 @@
+const { User } = require('../models/user.model');
+
+const getUser = async (req, res) => {
+  try {
+    const { userID } = req.user;
+    const user = await User.findById(userID).select('_id email firstName lastName');
+    res.json({ success: true, user });
+  } catch (error) {
+    res.json({ success: false, message: 'Error retrieving user data', errorMessage: error.message });
+  }
+};
+
+/**
+ * TODO:
+ * 1. Find and update user data
+ */
+const updateUserData = async (req, res) => {
+  try {
+    const { userID } = req.user;
+    const user = await User.findById(userID);
+    res.json({ success: true, user });
+  } catch (error) {
+    res.json({ success: false, message: 'Error updating user data', errorMessage: error.message });
+  }
+};
+
+/**
+ * TODO:
+ * 1. Find and delete user
+ */
+const deleteUser = async (req, res) => {
+  try {
+    const user = await User.find({});
+    res.json({ success: true, user });
+  } catch (error) {
+    res.json({ success: false, message: 'Error deleting user', errorMessage: error.message });
+  }
+};
```

Comment:
hey pranshu, you forgot to add functionality to `updateUserData` and `deleteUser` controllers.